.travis.yml

```diff
@@ -2,10 +2,19 @@ language: go
 go:
```

Comment:
Should we maybe drop 1.8 from the build matrix?

Reply:
I don't mind, I left it as it was being used by the original but yeah, two versions might be enough